server/src/main/java/com/google/sps/utils/AdRowProcessor.java

```diff
@@ -79,26 +91,112 @@ public static long getImpressionsMin(String impressionsField) throws IllegalArgu
     return 0;
   }
 
+  /*
+   * getImpressionsMax tries to parse the maximum number of impressions and throws an error
+   * if the field is not parsable.
+   */ 
   public static long getImpressionsMax(String impressionsField) throws IllegalArgumentException, NumberFormatException {
     String[] impressionsArray = formatImpressionField(impressionsField); 
     return Long.parseLong(impressionsArray[impressionsArray.length - 1]);   
   }
 
+  /* 
+   * Helper function for getImpressionsMin and getImpressionsMax.
+   * Min and max impression fields contain letters k and M as well as ≤. The 
+   * formatImpressionField helper method makes the fields easier to parse later.
+   */ 
+  public static String[] formatImpressionField(String str) {
+    String impressionsString = str.replace("k", "000").replace("M", "000000").replace(" ","").replace("≤", "");
+    String[] impressionsArray = impressionsString.split("-");
+    return impressionsArray; 
+  }
+
+  /* 
+   * getAgeTargets determines whether or not the ad is targeting age.
+   */ 
   public static boolean getAgeTargets(String str) {
```

Comment:
think a better name for this helper might be `isAgeTargeting` or something with `is` instead of `get` because it's basically just a bool test.